File: 251004/Yelkin/4-5/TopicService/src/main/java/by/yelkin/TopicService/dto/CommentRequestTo.java
package by.yelkin.TopicService.dto;

import jakarta.validation.constraints.Size;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class CommentRequestTo {
    private Long id;
    @Size(min = 2, max = 2048, message = "Content must be between 2 and 2048 characters")
    private String content;
    private Long topicId;

    public CommentRequestTo(Long id) {
        this.id = id;
    }
}

File: 251003/Kukhatskavolets/microblog/src/main/kotlin/by/kukhatskavolets/dto/requests/CommentRequestTo.kt
package by.kukhatskavolets.dto.requests

import jakarta.validation.constraints.Size

data class CommentRequestTo(
    val id: Long = 0,
    val tweetId: Long = 0,

    @field:Size(min = 2, max = 2048)
    val content: String = ""
)